plugins/gateway/kong/plugin.go

```diff
@@ -65,13 +73,21 @@ func createTelemetry(kong *pdk.PDK) (*models.Telemetry, error) {
 	truncatedBodyReq := false
 	truncatedBodyRes := false
 
+	requestTime, err := getRequestTimeFromContext(kong)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get request time from context %v", err)
+	}
+	responseTime, err := common.GetTimeNowRFC3339Nano()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get time now: %v", err)
+	}
 	routedService, err := kong.Router.GetService()
 	if err != nil {
 		return nil, fmt.Errorf("failed to get routed serivce: %v", err)
 	}
-	clientIP, err := kong.Client.GetIp()
+	clientIP, err := kong.Client.GetForwardedIp()
```

Comment:
We don't want to use `GetIp()` if failed to get (or `ForwardedIp` is empty?)

Reply:
Not sure it is needed. From what I saw, it will always return an IP even if nothing is configured and the IP is the IP of the load-balancer/node,

Reply:
Unless the docs claims that `GetForwardedIp()` always return an IP, I think we can just add it to be safe. Maybe someone will deploy it differently.

Reply:
> // kong.Client.GetForwardedIp() returns the remote address of the client making the request.
> // Unlike kong.client.get_ip, this function will consider forwarded addresses in cases
> // when a load balancer is in front of Kong. Whether this function returns a forwarded
> // address or not depends on several Kong configuration parameters:
> //
> //   - trusted_ips
> //   - real_ip_header
> //   - real_ip_recursive

The important thing being:
_

> Unlike kong.client.get_ip, this function will consider forwarded addresses in cases
> when a load balancer is in front of Kong.

_

Consider sounds to me like it does the same as GetIP + considering few other things

Reply:
sounds good